benchmark/buffers/buffer-iterate.js

```diff
@@ -29,20 +29,23 @@ function main(conf) {
 function benchFor(buffer, n) {
   bench.start();
 
-  for (var k = 0; k < n; k++)
-    for (var i = 0; i < buffer.length; i++)
+  for (var k = 0; k < n; k++) {
```

Comment:
@mscdex To make the decision about things like that, I looked at surrounding code and also used my own judgment as to what was more readable.

In this case, having one `for` with braces wrapping another `for` without braces seemed less readable to me than both having braces. Additionally, there were no other examples in this file of blocks without braces.